Plugins/GitHub3/GitHub3Plugin.cs

```diff
@@ -72,12 +72,15 @@ public static string OAuthToken
     public class GitHub3Plugin : GitPluginBase, IRepositoryHostPlugin
     {
         private readonly TranslationString _viewInWebSite = new("View in {0}");
-        private readonly TranslationString _tokenAlreadyExist = new("You already have an OAuth token. To get a new one, delete your old one in Plugins > Settings first.");
+        private readonly TranslationString _tokenAlreadyExist = new("You already have an personal access token. To get a new one, delete your old one in Plugins > Plugin Settings first.");
+        private readonly TranslationString _generateToken = new("Generate a GitHub personal access token");
+        private readonly TranslationString _manageToken = new("Manage GitHub personal access token");
+        private readonly TranslationString _openLinkFailed = new TranslationString("Fail to open the link");
```

Comment:
Should the English.Plugins.xlf be updated? 🤔

Reply:
That surprised me also but I launched to localization command and none of the string were translated. The build doesn't complained also.
So I think that the plugins are not translated....